tpot/base.py

```diff
@@ -890,28 +890,31 @@ def _evaluate_individuals(self, individuals, features, target, sample_weight=Non
                 eval_individuals_str.append(individual_str)
                 sklearn_pipeline_list.append(sklearn_pipeline)
 
+        # make partial function
+        partial_wrapped_cross_val_score = partial(_wrapped_cross_val_score,
+                            features=features,
+                            target=target,
+                            cv=self.cv,
+                            scoring_function=self.scoring_function,
+                            sample_weight=sample_weight,
+                            max_eval_time_mins=self.max_eval_time_mins,
+                            groups=groups
+                            )
+
         # evalurate pipeline
         resulting_score_list = []
         # chunk size for pbar update
         for chunk_idx in range(0, len(sklearn_pipeline_list), self.n_jobs * 4):
-            jobs = []
-            for sklearn_pipeline in sklearn_pipeline_list[chunk_idx:chunk_idx + self.n_jobs * 4]:
-                job = delayed(_wrapped_cross_val_score)(
-                    sklearn_pipeline=sklearn_pipeline,
-                    features=features,
-                    target=target,
-                    cv=self.cv,
-                    scoring_function=self.scoring_function,
-                    sample_weight=sample_weight,
-                    max_eval_time_mins=self.max_eval_time_mins,
-                    groups=groups
-                )
-                jobs.append(job)
-            parallel = Parallel(n_jobs=self.n_jobs, verbose=0, pre_dispatch='2*n_jobs')
-            tmp_result_score = parallel(jobs)
-
+            if self.n_jobs == 1:
+                tmp_result_scores = [partial_wrapped_cross_val_score(sklearn_pipeline=sklearn_pipeline)
+                for sklearn_pipeline in sklearn_pipeline_list[chunk_idx:chunk_idx + self.n_jobs * 4]]
```

Comment:
This line should be indented more to make it clear that it's a continuation of the previous line.